pkgs/sops-install-secrets/nixos-test.nix

```diff
@@ -36,7 +36,7 @@
 
       users.users.example-user = {
         isNormalUser = true;
-        passwordFile = config.sops.secrets.test_key.path;
+        hashedPasswordFile = config.sops.secrets.test_key.path;
```

Comment:
For the 22.03 test we need to make this conditional.

Reply:
Ah yes, of course. I've added a fixup [b43feda](https://github.com/Mic92/sops-nix/pull/433/commits/b43feda77e83656972d2b5e8d4c0d36f74cd8618).